guides/common/modules/proc_optimizing-performance-by-removing-nics-from-database.adoc

```diff
@@ -0,0 +1,37 @@
+:_content-type: PROCEDURE
+
+[id="proc_filtering-and-removing-facts_{context}"]
+[id="proc_optimizing-performance-by-removing-nics-from-database_{context}"]
+= Optimizing performance by removing NICs from database
+
+Filter and exclude the connections using the `Exclude pattern for facts stored in {Project}` and  `Ignore interfaces with matching identifier` option.
+By default, these options are set to a reasonable list for hypervisors such as, `libvirt`, `ovirt`, `RHV`, or `VMWare`.
+If you name the virtual interfaces differently, you may update this filter to use it according to your requirements.
+
+.Procedure
+
+. Login to your {Project} instance.
+
+.. Open *Administer* > *Settings* > *Facts*.
+.. To filter out all interfaces starting with specific names, for example, `blu`, add `blu*` to the `Ignore interfaces with matching identifier` option.
+.. To prevent databases from storing facts related to interfaces starting with specific names, for example, `blu`, add `blu*` to the `Exclude pattern for facts stored in {Project}` option.
++
+By default, it contains the same list as the `Ignore interfaces with matching identifier` option.
+You can override it based on the requirements.
+This filters out facts completely without storing them.
+
+. To optimize performance and cleanup facts and interfaces matching filters set above, open terminal:
+
+.. To remove matching facts from the database (as added in *Administer* > *Settings* > *Facts* > the `Exclude pattern for facts stored in {Project}` option):
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# foreman-rake facts:clean
+----
+
+.. To remove matching interfaces from the database (as added in *Administer* > *Settings* > *Facts* > the `Ignore interfaces with matching identifier` option):
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# foreman-rake interfaces:clean
+----
```

Comment:
is this file a duplicate? Please fix everything I've suggested for the first file in here as well.